weed/filer/filerstore_wrapper.go

```diff
@@ -282,7 +282,8 @@ func (fsw *FilerStoreWrapper) prefixFilterEntries(ctx context.Context, dirPath u
 	}
 
 	var notPrefixed []*Entry
-	lastFileName, err = actualStore.ListDirectoryEntries(ctx, dirPath, startFileName, includeStartFile, limit, func(entry *Entry) bool {
+	fetchLimit := int64(500)
```

Comment:
may be make constant ?

Reply:
OK，i will set a const

Reply:
how about only set to `FetchLimit` if `limit < FetchLimit`?

Reply:
> how about only set to `FetchLimit` if `limit < FetchLimit`?

I suppose the best number can only found by benchmark , but set by parameter limit here is not a good idea ,as i mentioned : The limit parameter supposed to limit the number of results and should not affect the number of entries obtained each time.